addons/website/views/snippets/s_carousel.xml

```diff
@@ -56,14 +56,14 @@
                 </div>
             </div>
             <!-- Controls -->
-            <a class="carousel-control-prev" href="#myCarousel" data-slide="prev" role="img" aria-label="Previous" title="Previous">
+            <div class="carousel-control-prev o_not_editable" contenteditable="false" href="#myCarousel" data-slide="prev" role="img" aria-label="Previous" title="Previous">
```

Comment:
@younn-o You changed leaving the href... but you left the div... I guess it's a mistake? I'll try and fix on my own.